delfin/drivers/dell_emc/vmax/client.py

```diff
@@ -226,3 +226,31 @@ def clear_alert(self, sequence_number):
         """Clear alert for given sequence number."""
         return self.rest.clear_alert(sequence_number, version=self.uni_version,
                                      array=self.array_id)
+
+    def get_array_performance_metrics(self, storage_id, interval):
+        """Get performance metrics."""
+        try:
+            # Fetch VMAX Array Performance data from REST client
+            # TODO  :
+            #  Check whether array is registered for performance collection
+            #  in unisphere
+            perf_data = self.rest.get_array_performance_metrics(
+                self.array_id, interval)
+            # parse VMAX REST response to metric->values map
+            metrics_value_map = perf_utils.parse_performance_data(perf_data)
+            # prepare  labels required for array_leval performance data
+            labels = {'storage_id': storage_id, 'resource_type': 'array'}
```

Comment:
driver logic may be differnt in preparing labels, for different resources , different label keys and different ways of filling them.. For array it may look same, but in generic it is driver specific..

Reply:
then these should be put in constants